code-review/README.md

```diff
@@ -1,76 +1,101 @@
+  - Do not squash until the branch is ready to merge.
+  - Reviewers should be able to read individual updates based on their earlier feedback.
+
+- **Seek to understand the reviewer's perspective**
+- **Try to respond to every comment**
+- **Wait to merge the branch until continuous integration tells you the test suite is green in the branch**
+  - TDDium, Travis CI, CircleCI, Github Actions, etc.
+
+- **Merge once you feel confident in the code and its impact on the project**
+- **Final editorial control rests with the pull request author**
 
 ## Reviewing Code
 
-Understand why the change is necessary (fixes a bug, improves the user
-experience, refactors the existing code). Then:
-
-- Communicate which ideas you feel strongly about and those you don't.
-- Identify ways to simplify the code while still solving the problem.
-- If discussions turn too philosophical or academic, move the discussion offline
-  to a regular Friday afternoon technique discussion. In the meantime, let the
-  author make the final decision on alternative implementations.
-- Offer alternative implementations, but assume the author already considered
-  them. ("What do you think about using a custom validator here?")
-- Seek to understand the author's perspective.
-- Sign off on the pull request with a 👍 or "Ready to merge" comment.
-- Remember that you are here to provide feedback, not to be a gatekeeper.
+Understand why the change is necessary (fixes a bug, improves the user experience, refactors the existing code).
+
+Then:
+
+- **Communicate which ideas you feel strongly about and those you don't**
+- **Identify ways to simplify the code while still solving the problem**
+- **If discussions turn too philosophical or academic, move the discussion offline to a regular Friday afternoon technique discussion**
+  - In the meantime, let the author make the final decision on alternative implementations.
+
+- **Offer alternative implementations**
+  - But assume the author already considered them.
+  - "What do you think about using a custom validator here?"
+
+- **Seek to understand the author's perspective**
+- **Sign off on the pull request with a 👍 or "Ready to merge" comment**
+- **Remember that you are here to provide feedback, not to be a gatekeeper**
 - When suggesting changes using the "Add a suggestion" feature:
-  - Communicate clearly which lines you suggest adding/removing.
-  - Test the suggested changes to validate it works whenever possible.
-  - When not possible, let the pull request author know that you did not test the suggestion.
-  - Provide some context to let the author know why you're suggesting the change.
+  - **Communicate clearly which lines you suggest adding/removing**
+  - **Test the suggested changes to validate it works whenever possible**
+  - **When not possible, let the pull request author know that you did not test the suggestion**
+  - **Provide some context to let the author know why you're suggesting the change**
```

Comment:
For this section to be consistent with others, should the parent bullet be the one that's bolded?

Reply:
I think it's an exception, "When suggesting changes using the "Add a suggestion" feature:" isn't an advice or an action and we highlight advices / actions